src/osp/Resource/ImporterData.h

```diff
@@ -83,7 +84,12 @@ struct ImporterData
 struct Prefabs
 {
     // [prefab Id][object]
-    lgrn::IntArrayMultiMap<int, int>    m_prefabs;
+    lgrn::IntArrayMultiMap<int, int>    m_prefabs{};
+    lgrn::IntArrayMultiMap<int, int>    m_prefabParents{};
```

Comment:
is the `{}` needed to get the initialization behavior you want?

The default constructor would be called either way, no?

Reply:
I think I put it there to try to fix an error, but the error wasn't actually there?